deeppavlov/models/kbqa/wiki_parser_online.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2017 Neural Networks and Deep Learning lab, MIPT
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from logging import getLogger
+from typing import List, Tuple, Dict
+
+import requests
+
+from deeppavlov.core.common.registry import register
+
+log = getLogger(__name__)
+
+
+def get_answer(query: str) -> List[Dict[str, Dict[str, str]]]:
+    url = 'https://query.wikidata.org/bigdata/namespace/wdq/sparql'
+    data = []
+    for i in range(5):
+        try:
+            data_0 = requests.get(url, params={'query': query, 'format': 'json'},  timeout=0.5).json()
```

Comment:
I think it would be nice if we could set target url, request timeout and allowed attempts count as init parameters for the `WikiParserOnline` class, setting them by default to `'https://query.wikidata.org/bigdata/namespace/wdq/sparql'`, `0.5` and `5` respectively.

Reply:
Moved target url to kbqa_cq_online.json